### ISSUE TITLE
other ways to contact you

### ISSUE THREAD

**Issue by rothos** [Project-Evidence/project-evidence.github.io#16]
I saw it also mentioned as a comment in one of the pull requests, but seemed like it deserved a ticket here too.

If I didn't happen to know how Github Pages works I wouldn't have been able to contact you here. Get a protonmail address or something. Also, would be good to include a Github repo link in the document.

**Comment by EXPOSING-THE-LIKELY-WUHAN-LAB-LEAK** [comment]
I authored https://exposing-the-likely-wuhan-lab-leak.github.io/ in early April. 

If have any questions about that topic, can ask Me at: MyEnglishFriend@gmail.com

**Comment by davidfregoli** [comment]
The easiest way would probably to use something like [this](https://fire.fundersclub.com/)

**Comment by Project-Evidence** [comment]
Hello,

We appreciate your concern. However, we must balance our transparency with our desire to stay anonymous. We believe that it is easier to avoid being swayed by bias if our document cannot be linked to our real-life names.

In [Revision 2](project-evidence.github.io) we have included a link to the repository. We have made it more clear that one should open an issue to get in contact with us. We think opening a GitHub issue is a technical hurdle most new users can get around, especially if they have prior experience on forums such as Reddit.

Thank you!
